hoomd/md/pair/pair.py

```diff
@@ -452,6 +452,95 @@ def __init__(self, nlist, default_r_cut=None, default_r_on=0., mode='none'):
         # correctly?
 
 
+class ExpandedLJ(Pair):
+    r"""Expanded Lennard-Jones pair potential.
+
+    Args:
+        nlist (`hoomd.md.nlist.NList`): Neighbor list.
+        default_r_cut (float): Default cutoff radius :math:`[\mathrm{length}]`.
+        default_r_on (float): Default turn-on radius :math:`[\mathrm{length}]`.
+        delta (float): The horizontal shift of pair's interaction :math:`[\mathrm{length}]`.
+        mode (str): Energy shifting mode.
+
+    `ExpandedLJ` specifies that a horizontally shifted Lennard-Jones
+    type pair potential should be applied between every non-excluded
+    particle pair in the simulation.
+
+    .. math::
+        :nowrap:
+
+        \begin{eqnarray*}
+        V_{\mathrm{ExpandedLJ}}(r)  = & 4 \varepsilon \left[ \left(
+                                \frac{\sigma}{r - \Delta} \right)^{12} -
+                                \left( \frac{\sigma}{r - \Delta}
+                                \right)^{6} \right]; & r < (r_{\mathrm{cut}}) \\
+                             = & 0; & r \ge (r_{\mathrm{cut}}) \\
+        \end{eqnarray*}
+
+    See `Pair` for details on how forces are calculated and the
+    available energy shifting and smoothing modes.
+
+    Attention:
+        Due to the way that `ExpandedLJ` modifies the cutoff criteria, a smoothing mode
+        of *xplor* is not supported.
+
+    Set the ``max_diameter`` property of the neighbor list object to the largest
+    particle diameter in the system (where **diameter** is a per-particle
+    property of the same name in `hoomd.State`). TODO: check if needed this warning
+
+    Warning:
+        Failure to set ``max_diameter`` will result in missing pair
+        interactions.
```

Comment:
Might not need this either

Reply:
I think this should be left here so users don't forget to set the `max_diameter` in the system.

Reply:
No, remove this. Users must explicitly set the per type pair `r_cut` appropriately. 

You could add a note that if users want to replicate the HOOMD v2 SLJ behavior, they should set `r_cut = r_cut_unshifted + delta`.

Reply:
I think we should add a note in the documentation that the type pair r_cut should be set to reflect the delta shifts. That was not clear to me as I did my review.

Reply:
I added a note; see if this helps